src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -193,5 +203,10 @@ private void OnEngineChanged(object sender, System.Windows.Controls.SelectionCha
                 UpdateScript(editText.Text);
             }
         }
+
+        private void OnScriptEditorWindowClosed(object sender, EventArgs e)
+        {
+            nodeModel.CodeMigrated -= OnNodeModelCodeMigrated;
```

Comment:
I know it might seem overkill, but can you also unsubscribe the Closing event from this same method?